templates/components/common/navigation-list-custom.html

```diff
@@ -0,0 +1,101 @@
+{{#contains name "(dropdown)"}}
+    <a 
+        class="js-nav-pages-action c-navigation__link has-sub-menu{{#if is_active}} is-active{{/if}}{{#if active_category_navigation '===' 'true'}} is-active{{/if}}"
+        href="{{url}}"
+        data-collapsible="nav-pages-{{id}}"
+    >
+        {{strReplace name "(dropdown)" ""}}
+    
+        <svg xmlns="http://www.w3.org/2000/svg" width="25" height="24" viewBox="0 0 25 24" fill="none">
+            <mask id="mask0_2973_30873" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="0" y="0" width="25" height="24">
+                <path d="M6.5 9L12.5 15L18.5 9" stroke="black" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+            </mask>
+            <g mask="url(#mask0_2973_30873)">
+                <rect x="0.5" width="24" height="24" fill="{{#if is_active}}#5A64C1{{else}}#272727{{/if}}"/>
+            </g>
+        </svg>
+    </a>
+    <div class="c-navigation__megamenu" id="nav-pages-{{id}}" aria-hidden="true" tabindex="-1">
+        <div aria-hidden="true" class="js-quick-search">
+            {{> components/common/quick-search name='nav-menu-quick-search'}}
+        </div>
+        <ul class="c-navigation__submenu">
+            {{#each categories}}
+                {{#each children}}
+                    <li>
+                        {{#if children}}
+                            <a 
+                                class=" js-nav-pages-action has-sub-menu{{#if is_active}} is-active{{/if}} c-navigation__link c-navigation__submenu-header"
```

Comment:
Similar here, `c-navigation__link`  & `c-navigation__submenu-header` shouldn't be applied to the same element because both classes are on the same `component` level and may overwrite each other